test/moduleId.html

```diff
@@ -5,6 +5,9 @@
 		<title>QUnit ModuleId Test Suite</title>
 		<link rel="stylesheet" href="../dist/qunit.css">
 		<script src="../dist/qunit.js"></script>
+		<script>
+			QUnit.warning.off();
+		</script>
```

Comment:
this should not happen here and other minor and specifc tests files, unless they intentionally test a deprecated feature.